wallet-new/src/Cardano/Wallet/WalletLayer/Kernel/Conv.hs

```diff
@@ -208,13 +212,20 @@ data InvalidRedemptionCode =
   | InvalidRedemptionCodeInvalidBase58 Text
 
     -- | AES decryption error (for paper wallets)
-  | InvalidRedemptionCodeCryptoError CryptoError
+  | InvalidRedemptionCodeCryptoError Text
```

Comment:
To avoid orphan instances for the `cryptonite` `CryptoError`… which is perhaps unnecessarily ideological as we're giving up functionality?
https://input-output-rnd.slack.com/archives/G8PHF04B0/p1535541556000100

Similar thing going on with [TimeExecutionLimit](https://github.com/input-output-hk/cardano-sl/blob/de6e30deb05a70c7aed682a9123c9e8bd9a14422/wallet-new/src/Cardano/Wallet/WalletLayer/ExecutionTimeLimit.hs#L24-L35), but there I did custom instances.

Reply:
Borderline similar thing going in with #3493 too, but that one uses internal types and is more tricky.

Reply:
Hmmmm.. Okay.